src/components/ResourceOverview/AmbulanceOverviewPage.tsx

```diff
@@ -0,0 +1,303 @@
+    const ambulanceId = selectedAmbulance;
+    deleteAmbulance({ variables: { id: ambulanceId } });
+    setOpenModal(false);
+  };
+
+  const open = Boolean(anchorEl);
+  const id = open ? 'simple-popper' : undefined;
+
+  const paraStyle = pStyles();
+  const classes = useLayout();
+  const hRow = headerRow();
+  const table = tableStyles();
+  const dRow = dataRow();
+  const optionBtn = options();
+  const dialogStyle = dialogStyles();
+
+  const cells = ambulances.map((ambulance: Ambulance) => {
+    return (
+      <TableRow key={ambulance.id}>
+        <TableCell classes={{ root: dRow.root }}>
+          {`#${ambulance.vehicleNumber}`}
+        </TableCell>
+        <TableCell classes={{ root: optionBtn.root }}>
+          <IconButton data-id={ambulance.id} onClick={handleClickOptions}>
+            <MoreHorizIcon style={{ color: Colours.Black }} />
+          </IconButton>
+        </TableCell>
+      </TableRow>
+    );
+  });
+
+  return (
+    <div className={classes.wrapper}>
+      <Typography variant="h5">Ambulance Overview</Typography>
+      <Typography variant="body2" classes={{ body2: paraStyle.body2 }}>
+        A list of all ambulances that can be added to an event.
+      </Typography>
+      <TableContainer>
+        <Table classes={{ root: table.root }}>
+          <TableHead>
+            <TableRow>
+              <TableCell classes={{ root: hRow.root }}>
+                Ambulance number
+              </TableCell>
+              <TableCell />
+            </TableRow>
+          </TableHead>
+          <TableBody>
+            {cells}
+            <Popper
+              id={id}
+              open={open}
+              popperOptions={{
+                modifiers: { offset: { enabled: true, offset: '-69.5,0' } },
+              }}
+              anchorEl={anchorEl}
+            >
+              <div>
+                <Table className={classes.tablePopper}>
+                  <TableBody>
+                    <TableRow
+                      hover
+                      classes={{ hover: optionBtn.menuHover }}
+                      onClick={handleClickEdit}
+                    >
+                      <TableCell classes={{ root: optionBtn.menuCell }}>
+                        <Typography variant="body2">Edit</Typography>
+                      </TableCell>
+                    </TableRow>
+                    <TableRow hover onClick={() => setOpenModal(true)}>
+                      <TableCell classes={{ root: optionBtn.menuDelete }}>
+                        <Typography variant="body2">Delete</Typography>
+                      </TableCell>
+                    </TableRow>
+                  </TableBody>
+                </Table>
+              </div>
+            </Popper>
+          </TableBody>
+        </Table>
+      </TableContainer>
+
+      <Dialog classes={{ paper: dialogStyle.paper }} open={openModal}>
+        <DialogTitle classes={{ root: dialogStyle.dialogTitle }}>
+          <Typography variant="h6">
```

Comment:
I think this is causing the warning in the console `Warning: validateDOMNesting(...): <h6> cannot appear as a child of <h2>.` Either don't use DialogTitle or see if you can change DialogTitle to use the "h6" variant.